app/assets/stylesheets/admin/components/jquery_dialog.scss

```diff
@@ -42,21 +42,25 @@ light: #ccc
 .ui-dialog .ui-corner-all{
   border-radius: 8px;
 }
-.ui-dialog  {
-  .ui-state-hover, .ui-state-focus{
-    border: none;
-    background: none;
-    color: #545454;
-  }
-}
+// .ui-dialog  {
+//   .ui-state-hover, .ui-state-focus{
+//     border: none;
+//     background: none;
+//     color: #545454;
+//   }
+// }
```

Comment:
Is this WIP?

Reply:
Nope, looks like it wasn't cleaned up. Added to my list.